TheMovieDb/Networking/Request.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Request.swift
+//  TheMovieDb
+//
+//  Created by Sandra Herrera on 28/10/21.
+//
+
+import Foundation
+
+
+
+class NetworkManager {
```

Comment:
`final` class here
also let's remove the 3 spaces after import

do we need a class for this Network layer ?